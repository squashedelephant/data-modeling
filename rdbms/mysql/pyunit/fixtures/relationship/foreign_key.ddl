CREATE TABLE league(id INT NOT NULL AUTO_INCREMENT, name CHAR(15) NOT NULL, PRIMARY KEY (id));
CREATE TABLE baseball(id INT NOT NULL AUTO_INCREMENT, league_id INT, team VARCHAR(30), city VARCHAR(30), state VARCHAR(2), PRIMARY KEY (id), INDEX league_idx (league_id), FOREIGN KEY (league_id) REFERENCES league(id) ON DELETE CASCADE);
