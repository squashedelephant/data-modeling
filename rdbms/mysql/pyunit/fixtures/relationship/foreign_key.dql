SELECT l.name, b.team, b.city, b.state FROM league AS l INNER JOIN baseball AS b WHERE l.id = b.league_id;
