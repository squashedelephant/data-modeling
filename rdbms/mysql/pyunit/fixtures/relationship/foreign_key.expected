National League
San Francisco Giants
San Francisco
CA
National League
Los Angeles Dodgers
Los Angeles
CA
American League
New York Yankees
New York
NY
American League
Boston Red Sox
Boston
MA
